dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -14,6 +14,22 @@ info:
     name: Apache License Version 2.0
     url: https://github.com/dockstore/dockstore/blob/develop/LICENSE
   version: 1.9.0-alpha.1-SNAPSHOT
+servers:
+- url: /api
```

Comment:
So this one will kick in if we for example create a perf.dockstore.net?

Reply:
My belief is yes, the first choice should kick in and work for any openapi.yaml that is hosted on a server at <domain-name>/api/openapi,yaml which it should be (?) for any thing we deploy using our compose setup/cloud formation approach.